### PR TITLE
write netplan directly to the target

### PR DESCRIPTION
**Pull request by mwhudson**
rather than via curtin and then cloud-init

### CODE REVIEW

**Comment by mwhudson** [comment]
For https://bugs.launchpad.net/subiquity/+bug/1869967

**Comment by vorlonofportland** [comment]
My only reservation is that I have a trace of a memory that we might have changed this intentionally to go via curtin, either for reasons of timing of when things are written out, or sanity of interfaces, or something.  Do we need to dig a bit on what the rationale was for moving to this approach, before we change it back?

**Comment by raharper** [comment]
> My only reservation is that I have a trace of a memory that we might have changed this intentionally to go via curtin, either for reasons of timing of when things are written out, or sanity of interfaces, or something. Do we need to dig a bit on what the rationale was for moving to this approach, before we change it back?

The change came from commit 36bcc539c39bf5083865f3c2273737e13455fb17 .  The commit message says moving to curtin config is a side-effect.

And previously it was written by curtin, including disabling cloud-init networking.

fb995c820918a0a995c8a63522d23d2c0ae6d1bb

File: subiquitycore/models/network.py [hunk 1]
```diff
@@ -350,7 +352,14 @@ def get_all_netdevs(self, include_deleted=False):
     def get_netdev_by_name(self, name):
         return self.devices_by_name[name]
 
-    def render(self):
+    def stringify_config(self, config):
+        return '\n'.join([
+            "# This is the network config written by '{}'".format(
```
Comment (raharper) [inline subiquitycore/models/network.py:357]: Maybe @vorlonofportland would like a specific breadcrumb here to indicate that this is _the_ source of truth for netplan config; that it is user editable, etc... maybe in the same form as it was written in the previous installer?

Comment (vorlonofportland) [reply subiquitycore/models/network.py:357]: FTR I'm ok with this as-is, I think breadcrumbs are most important when we need to communicate to a user /not/ to edit a thing and to look elsewhere.

**Pull request merged**